test/test_with_xpress.jl

```diff
@@ -1027,3 +1027,34 @@ end
 
 end
 
+@testset "PVspecs" begin
+    ## Scenario 1: Palmdale, CA; array-type = 0 (Ground-mount)
+    post_name = "pv.json" 
+    post = JSON.parsefile("./scenarios/$post_name")
+    scen = Scenario(post)
+    
+    print("\nlatitude: ", post["Site"]["latitude"])
+    print("\narray type: ", scen.pvs[1].array_type) # 0: Ground Mount Fixed (Open Rack); 1: Rooftop, Fixed
+    print("\ntilt: ", scen.pvs[1].tilt)
+    print("\nazimuth: ", scen.pvs[1].azimuth)
+ 
+    @test scen.pvs[1].tilt ≈ post["Site"]["latitude"] 
+    @test scen.pvs[1].azimuth ≈ 180
+
+    ## Scenario 2: Palmdale, CA; array-type = 1 (roof)
+    post["PV"]["array_type"] = 1 
+    scen = Scenario(post)
+
+    @test scen.pvs[1].tilt ≈ 10
+
+    ## Scenario 3:Cape Town; array-type = 0 (ground)
+    post["Site"]["latitude"] = -33.974732
+    post["Site"]["longitude"] = 19.130050
+    post["PV"]["array_type"] = 0 
+    scen = Scenario(post)
+
+    @test scen.pvs[1].tilt ≈ abs(post["Site"]["latitude"])
```

Comment:
It looks like we also need to test a user provided tilt value (and with your changes a user provided tilt would not be used because of the `tilt::Real= array_type == 1 ? 10 : abs(latitude)` in the PV constructor signature).

Reply:
added test in https://github.com/NREL/REopt.jl/pull/71/commits/8b35e5d34a0424cee6dec1c027e22c4681c5c1df